contract/src/lib.rs

```diff
@@ -336,15 +358,17 @@ impl BtcLightClient {
         btc_types::network::get_litecoin_config(self.network)
     }
 
-    #[cfg(feature = "zcash")]
-    pub fn get_config(&self) -> btc_types::network::ZcashConfig {
-        btc_types::network::get_zcash_config(self.network)
+    #[cfg(feature = "dogecoin")]
+    pub fn get_config(&self) -> btc_types::network::NetworkConfig {
```

Comment:
Let's move the get_config and get_network to seperate files, so we can avoid using flags here

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/101/commits/8d8c9246654c642beb87a51bb64210b87841563e